### DISCUSSION TITLE
Simple Radio App

### DISCUSSION THREAD

**Discussion by spykee2005** [BimmerGestalt/AAIdrive#562] [Q&A]
Hi all,

I was using Simple Radio App but it does not appear anymore on my IDrive app list like Apple Music or Spotify.

Preaviously I had access to it and was able to search, go to liste favorite and so on.

Your help would be much appreciated :-))

Thx!!!

**Comment by hufman** [comment]
I just tried and installed version 4.8.0 of [Simple Radio from the store](https://play.google.com/store/apps/details?id=com.streema.simpleradio), and it works for me, I can browse and search and everything.
Try rescanning the apps by pulling down in the Music tab in AAIdrive's phone interface.
